test/powershell/Language/Operators/TernaryOperator.Tests.ps1

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace System.Management.Automation
+using namespace System.Management.Automation.Language
+
+Describe "Ternary Operator" -Tags CI {
+    Context "Parsing of ternary operator" {
+        BeforeAll {
+            $testCases_1 = @(
```

Comment:
Can we use more informative name than "_1"?

Reply:
Updated to `testCases_basic` and `testCases_incomplete`.